### ISSUE TITLE
Bold, Italic don't work in WhatsApp and Gmail

### ISSUE THREAD

**Issue by theloneranger305** [D4rK7355608/GoogleProductSansFont#2]
The bold and italic fonts look the same as the regular font in WhatsApp and Gmail
![Screenshot_20211202-141950](https://user-images.githubusercontent.com/43410389/144390178-b24ae195-7b75-4109-b739-099dfc17cf99.png)
![Screenshot_20211202-142245](https://user-images.githubusercontent.com/43410389/144390189-59699459-7e5e-44a6-b985-3f1036a205a8.png)

**Comment by theloneranger305** [comment]
![Screenshot_20211202-151238](https://user-images.githubusercontent.com/43410389/144397219-7979bbd4-2943-4bca-9a8f-12ec4e54a02d.png)
Even this language, don't know which, doesn't look good

**Comment by VladWinner** [comment]
Italic style is not available because variable font `Roboto-Regular` does not contain italic axis (required for Android 12)

**Comment by theloneranger305** [comment]
What if I provide you with the ttfs for those? I've edited the base font to
be black, light and medium. Have all 18 typefaces. It would be great if you
could include those in your module

**Comment by mld9401** [comment]
When i try to write these two letters it shows me squares instead.

![Screenshot_20220419-123419_Notes Keep.png](https://user-images.githubusercontent.com/15183134/163994813-7ac9a8de-6881-4c3c-8c50-8f70a0808fae.png)

Language : Kabyle ( Latine Tamazight)

Can you fix it please?

**Comment by mld9401** [comment]
I have found a way to make this font look as it should be on Whatsapp (also fixing some characters issue showing squares instead like those i have previously reported "some Kabyle letters") but unfortunatly bold font still doesn't work with Gmail. Here are the steps :

1. Download this magisk module : 
[Font.Changer.-.Banjax.Notched.Bold.zip](https://github.com/D4rK7355608/GoogleProductSansFont/files/8761188/Font.Changer.-.Banjax.Notched.Bold.zip)

2. Extract the zip file
3. Remove Regular.ttf found in font folder
3. Download the latest GoogleProductSansFont.zip
4. Extract the zip file than move all the fonts found in font folder to the font folder in the file already downloaded in (1)
5. Compress all the files into .zip 
6. Flash it using Magisk

It's recomanded to use 7-zip for PC or MT Manager for Android.

![Screenshot_20220524-094654_WhatsApp~2](https://user-images.githubusercontent.com/15183134/169996268-5d97dc2e-8f3e-41de-9915-8ad4bf782057.png)
